### ISSUE TITLE
Dashboard not found when deployed through Kubeflow

### ISSUE THREAD

**Issue by supertetelman** [jacobtomlinson/jupyterlab-nvdashboard#22]
I'm deploying some RAPIDS containers through Kubeflow and the nvdashboard does not work properly.

When I attempt to view the dashboard I see `Sorry, /nvdashboard/GPU-Memory is not a valid page`. If I manually hit http://<mykubeflow>/notebook/kubeflow/dask/nvdashboard/GPU-Memory I see the dashboard.

I'm guessing somewhere we are using an absolute path to the nvdashboard and kubeflow and other similar deployment platforms require us to use relative paths.

**Comment by jacobtomlinson** [comment]
Thanks for raising this! I've thrown in #23 as a first attempt to fix it. It's interesting that it seems to be able to get the list of dashboards (a json file) but not the dashboards themselves. Hopefully this will make things consistent.

@supertetelman do you have any interest in testing this PR out for me before I merge it?

**Comment by supertetelman** [comment]
That seems to be working.

I'm occasionally seeing what looks to be a different issue now on the NVLink tabs. The dashboard occassionally shows a 500 error and the logs are showing this:

```
ERROR:tornado.application:Uncaught exception GET /NVLink-Throughput (127.0.0.1)
HTTPServerRequest(protocol='http', host='sae-npn-01:32720', method='GET', uri='/NVLink-Throughput', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/web.py", line 1699, in _execute
    result = await result
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/bokeh/server/views/doc_handler.py", line 55, in get
    session = yield self.get_session()
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/gen.py", line 742, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/bokeh/server/views/session_handler.py", line 77, in get_session
    session = yield self.application_context.create_session_if_needed(session_id, self.request)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/gen.py", line 735, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/tornado/gen.py", line 748, in run
    yielded = self.gen.send(value)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/bokeh/server/contexts.py", line 215, in create_session_if_needed
    self._application.initialize_document(doc)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/bokeh/application/application.py", line 178, in initialize_document
    h.modify_document(doc)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/bokeh/application/handlers/function.py", line 133, in modify_document
    self._func(doc)
  File "/root/jupyterlab-nvdashboard/jupyterlab_nvdashboard/apps/gpu.py", line 233, in nvlink
    for i in range(ngpus)
  File "/root/jupyterlab-nvdashboard/jupyterlab_nvdashboard/apps/gpu.py", line 233, in <listcomp>
    for i in range(ngpus)
  File "/root/jupyterlab-nvdashboard/jupyterlab_nvdashboard/apps/gpu.py", line 230, in <listcomp>
    for j in range(nlinks)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/pynvml/nvml.py", line 1999, in nvmlDeviceGetNvLinkUtilizationCounter
    check_return(ret)
  File "/opt/conda/envs/rapids/lib/python3.6/site-packages/pynvml/nvml.py", line 366, in check_return
    raise NVMLError(ret)
pynvml.nvml.NVMLError_NotSupported: Not Supported
ERROR:tornado.access:500 GET /NVLink-Throughput (127.0.0.1) 206.01ms
```

**Comment by jacobtomlinson** [comment]
Great! This fix will be released in `0.1.10` shortly!

If you continue to have the other problem you mentioned feel free to raise another issue.
